x-pack/plugin/esql/qa/testFixtures/src/main/resources/mv_percentile.csv-spec

```diff
@@ -0,0 +1,122 @@
+default
```

Comment:
Maybe at least one more test :-). Something that deals with constants in various not-so-used-not-so-common usecases, to prove our QL-side of optimizations are in place and we don't miss things:

`ROW a = [5, 5, 10, 12, 5000], n = null, b = 50 | eval evalNull = null / 2, evalValue = 31 + 1 | limit 1 | EVAL mv_percentile(b, 90), mv_percentile(a, b), mv_percentile(null, null), mv_percentile(null, b), mv_percentile(evalNull, b), mv_percentile(evalValue, b), mv_percentile(n, b)`